documentation/docs/guides/middlewares.md

```diff
@@ -8,9 +8,43 @@ including the ones from `chi` and `gorilla` can be used with Fuego! :fire:
 You can use them to add functionalities to your routes, such as logging,
 authentication, etc.
 
-## App-level middlewares
+Middlewares can be registered at 2 levels:
 
-You can add middlewares to the whole server using the `Use` method:
+- **Route middlewares**: applies only on registered routes, easily scopable to a specific group, or several routes.
+- **Global middlewares**: applies on every request, even non-matching routes (useful for CORS for example).
```

Comment:
```suggestion
- **Route middlewares**: apply only on registered routes, easily scopable to a specific group, or several routes.
- **Global middlewares**: apply on every request, even non-matching routes (useful for CORS for example).
```

Or


```suggestion
- **Route middlewares**: applied only on registered routes, easily scopable to a specific group, or several routes.
- **Global middlewares**: applied on every request, even non-matching routes (useful for CORS for example).
```

Reply:
I think this would be nice actually

```suggestion
- **Route middlewares**: applies only to registered routes, is scoped to a specific group, or several routes.
- **Global middlewares**: applied on every request, even non-matching routes (useful for CORS for example).
```